specification/communication/data-plane/SipRouting/preview/2024-11-15-preview/communicationservicessiprouting.json

```diff
@@ -398,9 +421,37 @@
           "items": {
             "type": "string"
           }
+        },
+        "callerIdOverride": {
+          "type": "string",
+          "description": "Gets or sets caller ID override of the route."
```

Comment:
Let's mention that it overrides what is set at runtime to document which one takes precedence (or the inverse, whatever the behavior is when both are set)